integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/jira/issues/CARBON15262JsonGsonFormatterTenantModeTest.java

```diff
@@ -56,7 +56,7 @@ public void serviceDeployment() throws Exception {
 		sqlFileLis.add(selectSqlFile("Offices.sql"));
 		deployService(serviceName, createArtifact(getResourceLocation() + File.separator + "dbs" + File.separator +
 		                                          "rdbms" + File.separator + "h2" + File.separator +
```

Comment:
The operator should precede the line. Shall we fix this in other places too.

Reply:
Done